frontend/__tests__/inquiry/inquiries_table.test.tsx

```diff
@@ -0,0 +1,162 @@
+// Disabling these ESLint rules for mocking the "dayjs" module
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+// Disabling to mock return value of "dayjs.tz.guess" function
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+import { useQuery } from "@tanstack/react-query"
+import { render, screen } from "@testing-library/react"
+import InquiriesTable from "../../src/components/Inquiries/InquiriesTable"
+import "@testing-library/jest-dom"
+
+import dayjs from "dayjs"
+
+const inquiries = [
+  {
+    id: "92bc71bf-f42c-4b56-b55d-fddaf4633550",
+    text: "How is your work-life balance?",
+    created_at: "2024-09-22T18:20:53.734830",
+  },
+  {
+    id: "736381f5-becf-4783-8f04-98250e69c0b3",
+    text: "Is communication effective?",
+    created_at: "2024-09-22T12:30:53.734830",
+  },
+  {
+    id: "df1e0d27-4f77-4d6b-b997-2b07d892db17",
+    text: "How satisfied are you with the work environment?",
+    created_at: "2024-09-21T09:20:53.734830",
+  },
+  {
+    id: "13f29f63-2793-49b5-a6a9-c26acda6651f",
+    text: "How do you feel about your current role?",
+    created_at: "2024-09-19T09:20:53.734830",
+  },
+  {
+    id: "f919b688-f9c6-49cb-87c0-4dab68a4a04c",
+    text: "How is feedback typically given and received within the team?",
+    created_at: "2024-09-22T09:20:53.734830",
+  },
+]
+
+// Mock dependencies
+jest.mock("@tanstack/react-query")
+
+describe("Inquiries Table", () => {
+  const renderComponent = () => render(<InquiriesTable />)
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it("should display empty table when there's no inquiries.", () => {
+    ;(useQuery as jest.Mock).mockReturnValue({
+      data: { data: [] },
+      isPending: false,
+    })
+
+    renderComponent()
+    expect(screen.getAllByRole("row").length).toBe(1) // just the header row
+  })
+
+  // it("should display skeleton text when data is loading.", async () => {
```

Comment:
what is this for?

Reply:
This a test that I was not able to replicate. I wasn't able to render `InquiriesTable` when useQuery is in `isPending` mode:

```
   (useQuery as jest.Mock).mockReturnValue({
      data: null,
      isPending: true,
    });
```

Reply:
yeah. if it's "isPending" that means the data hasn't been received yet so nothing would display.  Sometimes people show a spinner or something.  But that's out of scope for this ticket.  I think you can just delete this test.

Reply:
I think you are right, maybe this check should belong in UI test..?
Removed as suggested.